docs/src/components/pricing/PricingTable.tsx

```diff
@@ -79,84 +86,132 @@ interface PlanPriceProps {
 export function PlanPrice(props: PlanPriceProps) {
   const { plan } = props;
 
+  const { licenseType } = useLicenseType();
+  const annual = licenseType === 'Annual';
+  const planPriceMinHeight = 64;
+
   if (plan === 'community') {
     return (
-      <Box sx={{ display: 'flex', alignItems: 'center', mt: 1 }}>
-        <Typography variant="h4" component="div" fontWeight="bold" color="success.600">
-          $0
-        </Typography>
-        <Box sx={{ width: 5 }} />
-        <Typography variant="body2" color="text.secondary" sx={{ mt: '4px' }}>
-          – free forever
+      <React.Fragment>
+        <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center', mt: 1, mb: 4 }}>
+          <Typography
+            variant="h3"
+            component="div"
+            fontWeight="bold"
+            color="success.600"
+            sx={{ mt: 4.5 }}
+          >
+            $0
+          </Typography>
+        </Box>
+        <Typography variant="body2" color="text.secondary" textAlign="center">
+          Free forever!
         </Typography>
-      </Box>
+      </React.Fragment>
     );
   }
+
+  const monthlyDisplay = annual;
+
+  const priceUnit = monthlyDisplay ? '/ dev / month' : '/ dev';
```

Comment:
Furthermore, the Premium Annual plan feels too crowded with numbers which might be confusing:
<img width="286" alt="Screenshot 2023-02-23 at 17 29 23" src="https://user-images.githubusercontent.com/13808724/220969704-d572fe41-a509-4672-b392-7198481121c3.png">

Reply:
Also, these numbers do not add up: ($49 * 12) - 25% = $441
I understand that the difference comes from the rounding of the monthly price, but this seems like another reason to avoid monthly pricing given that it's billed annually anyway 🙂

Reply:
Agreed, we throw too many numbers at the reader all at once here. As an aside—how long is this "early bird special" going to last? It's been around for as long as I can remember now and is starting to feel not-so-early anymore. 😅

Reply:
I've had both of these same questions, too. 
For reference, this is what was in design (I've had the same point about monthly vs annual too). @samuelsycamore I was told the reason for early bird still being there is because the Premium plan features are still not fully fledged out enough to consider it full price? 

Does this screenshot feel clearer to you both? 
<img width="669" alt="Figmamockup" src="https://user-images.githubusercontent.com/106991024/221671268-1ae8a9f2-f166-465a-b527-5b282b111d59.png">

Reply:
Looks much cleaner! 👍

Reply:
Agreed, I think that's a big improvement!